configs/next_router_vuejs.json

```diff
@@ -19,7 +19,7 @@
     }
   ],
   "stop_urls": [
-    ".html"
+    "(?:(?<!.html)(?<!/))$"
```

Comment:
Are you sure we don't need to escape the `.`? This will allow pages like `/rendering-html` and `/rendering-html.html`

Reply:
oops, fixed in https://github.com/algolia/docsearch-configs/pull/3899